studio/components/layouts/AuthLayout.tsx

```diff
@@ -0,0 +1,43 @@
+import DashboardLayout from '../../components/layouts/DashboardLayout'
+import Loading from '../../components/utils/Loading'
+import Error from '../../components/utils/Error'
+import { fetchOpenApiSpec } from '../../lib/api'
+import { ReactElement } from 'react'
+
+export default function AuthLayout({
+  title,
+  children,
+}: {
+  title: string
+  children: ReactElement
+}) {
+  const { isLoading, error } = fetchOpenApiSpec()
+
+  if (isLoading) return <Loading />
+  if (error) return <Error />
+
+  const buildSidebar = () => {
+    return [
+      {
```

Comment:
since this isn't constructing anything, it can just be a const variable with the links

Reply:
Done.